yarn.lock

```diff
@@ -4545,6 +4560,10 @@ jest@23.4.1:
     import-local "^1.0.0"
     jest-cli "^23.4.1"
 
+jquery@x.*:
```

Comment:
I’m also concerned by the dependency on jQuery. My understanding was
that the Semantic UI React family didn’t use jQuery at all. Do you know
what’s going on here?